src/colorizer/Dataset.ts

```diff
@@ -25,6 +26,11 @@ export type FeatureData = {
   tex: Texture;
   min: number;
   max: number;
+  units?: string;
+};
+
+export type FeatureMetaData = {
```

Comment:
it looks weird to define this after the thing that uses it.  javascript is funny.